api/src/controllers/sample.ts

```diff
@@ -1,15 +1,21 @@
 import { Request, Response } from 'express';
+import pool from '../util/pool';
+import * as db from 'zapatos/db';
 
 /**
  * Sample controller.
  * @param req HTTP request.
  * @param res HTTP response.
- * @returns HTPP response.
+ * @returns HTTP response.
  */
-const sampleHealthCheck = (req: Request, res: Response): Response => {
-    req.log.info('Sample health check route called.');
+const sampleHealthCheck = async (req: Request, res: Response): Promise<Response> => {
+    req.log.debug('Sample health check route called.');
 
-    return res.status(200).json({ message: 'pong' });
+    // TODO remove - this is just an example of how Zapatos can be used.
+    // In the future all DB access will be performed in the repository layer
+    const allUsers = await db.select('users', db.all).run(pool);
```

Comment:
To confirm, the first parameter is the table name and the second parameter is a `WHERE` condition? And in this case, `db.all` just specifies no condition and gets all rows from the table?

Reply:
That is correct. `db.select` is a helper method to avoid re-writing some of the more common SQL queries. Alternatively we can use a format where we just pass in a full SQL query. I highly recommend reading the docs if you want to learn more about Zapatos